components/user-manager/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/UserBean.java

```diff
@@ -18,46 +18,58 @@
 package org.wso2.carbon.user.mgt.ui;
 
```

Comment:
I see lots of places in the diff where no new code modification is done but due to formatting they have come to the diff. Therefore when making PRs in future, please make sure only the code changes you have done are in the PR.